opentreemap/treemap/templates/treemap/map_feature_detail.html

```diff
@@ -267,58 +267,19 @@ <h4 class="address" id="map-feature-address">{{ feature.address_full }}</h4>
 {% localize off %}
 // Define options for mapFeature.init(), which will be called in the
 // type-specific template inheriting from this one (e.g. plot_detail.html)
-var mapFeatureOptions = {
-    config: otm.settings,
+window.mapFeature = {
```

Comment:
We should consider "namespacing" all of our globals under `window.otm` so segregate them from the hundreds of globals already under `window`.

It may also be a good idea to explain, in a comment, why we are using globals and that we are only using them for static data.